### PR TITLE
Fix CircleCI Contexts

### PR DESCRIPTION
**Pull request by zackproser**
This pull request was programmatically opened by the multi-repo-updater program. It should be adding the 'Gruntwork Admin' context to any Workflows -> Jobs nodes and should also be leaving the rest of the .circleci/config.yml file alone. 

 This PR was opened so that all our repositories' .circleci/config.yml files can be converted to use the same CircleCI context, which will make rotating secrets much easier in the future.

### CODE REVIEW

**Comment by brikis98** [comment]
No files changed in the PR?

**Comment by zackproser** [comment]
> No files changed in the PR?

Yes, at the end of the night I updated this status at the bottom of the PR description here: https://github.com/gruntwork-io/prototypes/pull/85 but realized I should have dropped a comment here. 

TLDR; I will look into and fix why this is happening. I suspect the logic for detecting that the correct contexts are already in place worked here, yet execution incorrectly proceeded to creating a branch and PR anyway. I wanted to leave it open until I could confirm, then patch the code, close this PR and re-run.

**Comment by zackproser** [comment]
Fixed in https://github.com/gruntwork-io/prototypes/pull/85/commits/cfcfffdcd8120b7d471238d406eb8b256cbeea97

**Comment by zackproser** [comment]
Preparing to re-run the batch that includes this repo now that the bugs have been fixed.
